Directory.Build.props

```diff
@@ -0,0 +1,17 @@
+<Project>
+    <PropertyGroup>
+        <TargetFramework>net7.0</TargetFramework>
```

Comment:
Can we use this opportunity to update to .NET 8 (for LTS)?

Reply:
yep